README.md

```diff
@@ -1,45 +1,124 @@
-# React Native Releases
-
-[![GitHub Issues](https://img.shields.io/github/issues/react-native-community/releases.svg)](https://github.com/react-native-community/releases/issues) ![Contributions welcome](https://img.shields.io/badge/contributions-welcome-orange.svg)
-
-Stay up-to-date with the release activities of [React Native](https://github.com/facebook/react-native/) by [watching](https://github.com/react-native-community/releases/subscription) for [status reports](https://github.com/react-native-community/releases/issues?q=is%3Aopen+is%3Aissue+label%3A%22release+status%22). Or you can follow along as the release notes are prepared and help reviewing the overall [changelog](https://github.com/react-native-community/releases/blob/master/CHANGELOG.md).
```

Comment:
I think we should keep this Intro part, and as first section after it a small "How to use the changelog generator" section like the one existing before.

It could be shaped somewhat closely to:

--

To use it first off `git clone` the repo, then `yarn` and finally you can use it like in the following example:

```bash
yarn generate -b v0.61.5 -c v0.62.0-rc.3 -r ../react-native -f ./CHANGELOG.md -t [GITHUB_TOKEN]
```

In particular:

- `-r` needs to be a local copy of the `react-native` repo, on master branch, up-to-date
- `-t` needs to be a Personal Access Token generated via GitHub token with `read` authorization for `public_repos`

--

Reply:
I think we should just leave the changelog part to the changelog doc, but perhaps we can find a good place in the next iteration to link to it directly from the README.